manoseimas/lobbyists/models.py

```diff
@@ -29,6 +29,26 @@ class Lobbyist(CrawledItem):
     def __unicode__(self):
         return self.name
 
+    def get_clients(self):
+        clients = []
+        for declaration in self.lobbyistdeclaration_set.all():
+            clients.extend(declaration.clients.all())
+        return clients
```

Comment:
This was the simplest possible solution. NO optimization. It worked.

Optimization would be welcome. In another iteration perhaps?